### DISCUSSION TITLE
Got TLE depending on the network environment

### DISCUSSION THREAD

**Discussion by abhi08-04** [cpeditor/cpeditor#1392] [Q&A] [answered]
Hello Community.......
I'm new to cp editor and having issue while using cp editor using the proxy settings of my college's internet ...every time when I'm on my college internet it and run a simple hello world code it shows TLE whereas when I use my mobile hotspot without proxy it runs fine. I tried setting up the proxy in Option>>Preferences>>Advanced>>Network Proxy but it didn't helped.....

**Comment by ouuan** [comment]
Running competitive programming solutions should not involve the Internet. And the network proxy setting is only used for the update checker and optionally for WakaTime.

> every time when I'm on my college internet it and run a simple hello world code it shows TLE whereas when I use my mobile hotspot without proxy it runs fine.

There could be some other factors and you accidentally attribute the result to the proxy setting. Please verify again.

**Reply by ouuan** [reply]
Or maybe it's due to some cloud-based anti-virus scanning.

**Comment by ouuan** [comment, marked as answer]
Random unexpected TLE usually involves anti-virus scanning. See [My code gets TLE on every first execution after compilation - FAQ](https://cpeditor.org/docs/faq/#my-code-gets-tle-on-every-first-execution-after-compilation).

**Comment by abhi08-04** [comment]
The issue is the editor only works when either i'm using personnel hotspot internet or the wifi is off....As soon as it switch to my college's internet which runs on proxy ...it start showing TLE......
https://jmp.sh/oBD2qa8T Here is a 50 sec video of my issue......

**Reply by ouuan** [reply]
Please read my reply above. Try "Run" instead of "Compile and Run" to verify that TLE only happens on first execution.

**Comment by ouuan** [comment]
Discussions are different from issues. You need to "mark as answer" and not close discussions.
